From mail@chrismalley.co.uk  Mon Oct 27 11:44:03 2008
From: Chris Malley <mail@chrismalley.co.uk>
Date: Sat, 25 Oct 2008 22:07:32 +0100
Subject: USB: usbtmc: Use explicit unsigned type for input buffer instead of char*
To: gregkh <gregkh@suse.de>
Cc: linux-kernel@vger.kernel.org
Message-ID: <1224968852.25054.3.camel@helix.beotel.net>


From: Chris Malley <mail@chrismalley.co.uk>

Silences compiler warning about comparison with 0x80, and type now matches the
corresponding _bulk_out function.

drivers/usb/class/usbtmc.c: In function ‘usbtmc_ioctl_abort_bulk_in’:
drivers/usb/class/usbtmc.c:163: warning: comparison is always false due to limited range of data type

Signed-off-by: Chris Malley <mail@chrismalley.co.uk>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/class/usbtmc.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/class/usbtmc.c
+++ b/drivers/usb/class/usbtmc.c
@@ -133,7 +133,7 @@ static int usbtmc_release(struct inode *
 
 static int usbtmc_ioctl_abort_bulk_in(struct usbtmc_device_data *data)
 {
-	char *buffer;
+	u8 *buffer;
 	struct device *dev;
 	int rv;
 	int n;
